runtime/near-vm-runner/src/near_vm_runner.rs

```diff
@@ -218,7 +218,7 @@ impl NearVmConfig {
 //  major version << 6
 //  minor version
 const VM_CONFIG: NearVmConfig = NearVmConfig {
-    seed: (2 << 10) | (1 << 6) | 2,
+    seed: (2 << 29) | (2 << 6) | 0,
```

Comment:
What's changed here and why? 
Please also update the comment above. 
Same for similar changes below.

Reply:
I amended the the commit message that [introduced the change](https://github.com/near/nearcore/pull/9302/commits/679c2f1ba6af1fef5f813695f719180c9e717b08) to answer the first question. This is the usual way for us to "clear" the cache today.

Comments updated.